### ISSUE TITLE
Refactor inline documentation to puppet strings (yard) format

### ISSUE THREAD

**Issue by jeffmccune** [sensu/sensu-puppet#757]
After #756 and #682 are merged, the inline documentation needs to be updated to the new `puppet strings` format.  This format is described at https://github.com/puppetlabs/puppet-strings/blob/master/README.md

Potentially out of scope, but helper methods for types and providers should be moved into the PuppetX namespace and have documentation added.

**Comment by alvagante** [comment]
@ghoneycutt  going to start with this, so that I learn better the module params ;-)
Can't self assign it, anyway.

**Comment by alvagante** [comment]
#792 merged, ticket can be closed.
